### PR TITLE
[HunyuanVideo-1.5-Diffusers-480p_t2v_distilled] Add tiled VAE decoder test

### LINKED ISSUE #4619: Bring up HunyuanVideo 1.5 VAE decoder (tiled variant)

### PR DESCRIPTION
**Pull request by kamalrajkannan78**
### Ticket

- closes https://github.com/tenstorrent/tt-xla/issues/4619

### Problem description

- Non tiled vae decoder hangs ([#4485](https://github.com/tenstorrent/tt-xla/issues/4485)). As a quick workaround, bringup tiled variant.

### What's changed

- Added test for the tiled variant.

### Checklist
- [x] verified the changes through local testing in N150

### Logs

- [may11_vae_decoder_tiled.log](https://github.com/user-attachments/files/27603836/may11_h_vae_decoder_tiled.log)

### CODE REVIEW

**Review by saiarthiraguram — APPROVED** [review]

File: tests/torch/models/Hunyuan_1_5/test_vae_decoder.py [hunk 1]
```diff
@@ -9,19 +9,40 @@
...
 import torch_xla.runtime as xr
 from infra import Framework, run_graph_test
+from loguru import logger
 
 from third_party.tt_forge_models.hunyuan_1_5.pytorch import ModelLoader, ModelVariant
 
 
-@pytest.mark.skip(
-    reason="hang on TT — investigation pending — https://github.com/tenstorrent/tt-xla/issues/4485"
+@pytest.mark.parametrize(
```
Comment (ppadjinTT) [inline tests/torch/models/Hunyuan_1_5/test_vae_decoder.py:16]: @claude what happens when you call this test without specifying the variant, what is the default behavior

Comment (kamalrajkannan78) [reply tests/torch/models/Hunyuan_1_5/test_vae_decoder.py:16]: With `@pytest.mark.parametrize`, calling the test without specifying a variant collects **both** parametrized cases and runs them as separate tests — there's no "default" variant; pytest runs every entry in the list.

Running:
`pytest tests/torch/models/Hunyuan_1_5/test_vae_decoder.py::test_vae_decoder`

will execute:
- `test_vae_decoder[non_tiled]` → currently `xfail` (hang, tt-xla#4485)
- `test_vae_decoder[tiled]`     → currently `xfail` (3D mask, tt-mlir#8362)

Comment (ppadjinTT) [reply tests/torch/models/Hunyuan_1_5/test_vae_decoder.py:16]: ok tnx, we can check this in

**Review by ppadjinTT — APPROVED** [review]

**Review by sdjukicTT — APPROVED** [review]
